pyfar/classes/audio.py

```diff
@@ -428,6 +422,16 @@ def __init__(self, data, frequencies, fft_norm=None, comment=None,
 
         _Audio.__init__(self, 'freq', comment, dtype)
 
+        self._VALID_FFT_NORMS = ["none"]
+
+        self._init_frequency_domain_data(data, frequencies, fft_norm)
+
+    def _init_frequency_domain_data(self, data, frequencies, fft_norm):
```

Comment:
This looks a little artificial to me... What about setting `self._VALID_FFT_NORMS` in the `fft_norm` setter and calling the `fft_norm` setter in the `__init__` functions of FrequencyData and Signal? This would allow for different values of  `self._VALID_FFT_NORMS` in FrequencyData and Signal despite the inheritance.